ctapipe/coordinates/disp.py

```diff
@@ -0,0 +1,135 @@
+"""
+Helper functions and components for handling coordinate transformations
+during origin reconstruction using the disp method.
+"""
+import warnings
+
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import AltAz, SkyCoord
+from astropy.table import Table
+
+from ctapipe.coordinates import MissingFrameAttributeWarning, TelescopeFrame
+
+from ..containers import ArrayEventContainer, ReconstructedGeometryContainer
+from ..core import Component
+
+__all__ = ["horizontal_to_telescope", "telescope_to_horizontal", "DispConverter"]
+
+
+def horizontal_to_telescope(
+    alt: u.Quantity, az: u.Quantity, pointing_alt: u.Quantity, pointing_az: u.Quantity
+):
+    """Transform coordinates form horizontal coordinates into TelescopeFrame"""
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", MissingFrameAttributeWarning)
+
+        horizontal_coord = SkyCoord(alt=alt, az=az, frame=AltAz())
+        pointing = SkyCoord(alt=pointing_alt, az=pointing_az, frame=AltAz())
+        tel_frame = TelescopeFrame(telescope_pointing=pointing)
+
+        tel_coord = horizontal_coord.transform_to(tel_frame)
+
+    return tel_coord.fov_lon.to(u.deg), tel_coord.fov_lat.to(u.deg)
+
+
+def telescope_to_horizontal(
+    lon: u.Quantity, lat: u.Quantity, pointing_alt: u.Quantity, pointing_az: u.Quantity
+):
+    """Transform coordinates from TelescopeFrame into horizontal coordinates"""
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", MissingFrameAttributeWarning)
+
+        pointing = SkyCoord(alt=pointing_alt, az=pointing_az, frame=AltAz())
+        tel_coord = TelescopeFrame(
+            fov_lon=lon, fov_lat=lat, telescope_pointing=pointing
+        )
+        horizontal_coord = tel_coord.transform_to(AltAz())
+
+    return horizontal_coord.alt.to(u.deg), horizontal_coord.az.to(u.deg)
+
+
+class DispConverter(Component):
+    """Convert (norm, sign) predictions into (alt, az) predictions"""
+
+    # This is a temporary solution for simulations using a single pointing position
+    pointing_altitude = u.Quantity
```

Comment:
This assigns the class `u.Quantity` as a class members of `DispConverter`, I think this is not what you want to do here (

Reply:
Oh, yes that's wrong. Would it be correct to use `traitlets.Int` here?

Reply:
Int is certainly wrong for altitude / azimuth. What's the point here?

Reply:
I meant float, sorry. As the pointing information is not yet loaded with TableLoader, I want to store singular values for `pointing_altitude` and `pointing_azimuth` as a temporary solution for it to work with simulations.

Reply:
This is only used if `predict()` is called. So, `pointing_altitude` and `pointing_azimuth` could also be parameters of `predict()`, if this makes more sense.

Reply:
I would say yes, this makes more sense. Even more sense would be to fill this info in table loader from the now merged obs-config. Should be an easy change and could be done in a PR towards `master` even.

Reply:
I changed it to be parameters for now. I can look into the necessary changes to table loader later.